packages/otelbin/src/app/og/route.tsx

```diff
@@ -0,0 +1,61 @@
+// SPDX-FileCopyrightText: 2023 Dash0 Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { ImageResponse, type NextRequest } from "next/server";
+import { calcNodes } from "~/components/react-flow/useClientNodes";
+import ParentsNode from "./ParentsNode";
+import JsYaml from "js-yaml";
+import { editorBinding } from "~/components/monaco-editor/editorBinding";
+import type { IConfig } from "~/components/react-flow/dataType";
+import { calcScale } from "~/lib/metadataUtils";
+
+export const runtime = "edge";
+
+const edgeWidth = 80;
+
+export async function GET(request: NextRequest) {
```

Comment:
**Question:** What do we need this route for? AFAICT this can be deleted?

Reply:
I just put it as a defualt og image, yes it can be deleted.